components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/inbound/InboundAuthenticationRequestProcessor.java

```diff
@@ -120,7 +120,8 @@ protected InboundAuthenticationResponse buildResponseForFrameworkLogin(InboundAu
         response.addParameters(InboundAuthenticationConstants.RequestProcessor.CALL_BACK_PATH,
                 getCallbackPath(context));
         response.addParameters(InboundAuthenticationConstants.RequestProcessor.RELYING_PARTY, getRelyingPartyId());
-
+        response.addParameters(InboundAuthenticationConstants.RequestProcessor.AUTH_TYPE, getName());
+        response.setRedirectURL(getCallbackPath(context));
```

Comment:
This seems wrong. redirectURL is where inbound servlet should redirect (e.g. commonauth or client). callBackPath is where commonauth shoud redirect after authentication (/samlsso or /oauth2/authorize).